Gogoanime/dist/metadata.json

```diff
@@ -13,8 +13,8 @@
     
   ],
   "version": "1.0.0",
-  "logo": "https://cdn.psychopast.tk/files/gogoanime-logo.png?key=zsBPy7m=t79yBOvQQF5gKZRoRcpoi+kn",
-  "thumbnail": "https://cdn.psychopast.tk/files/gogoanime-thumb.png?key=zsBPy7m=t79yBOvQQF5gKZRoRcpoi+kn",
+  "logo": "https://i.imgur.com/Awp1MUc.png",
+  "thumbnail": "https://i.imgur.com/MmCT2wf.png",

```

Comment:
can you use a promotional image or a screenshot of the website instead?

Reply:
sure :)